projects/storefrontlib/src/cms-components/organization/order-approval/order-approval-detail/order-approval-detail-form/order-approval-detail-form.component.html

```diff
@@ -60,34 +60,33 @@
           </div>
         </div>
       </form>
-
-      <ng-container *ngIf="!approvalFormVisible">
-        <div class="form-group row">
-          <div class="col-lg-4 col-md-12">
-            <a
-              [routerLink]="{ cxRoute: 'orderApprovals' } | cxUrl"
-              class="btn btn-block btn-secondary"
-              >{{ 'orderApproval.back' | cxTranslate }}</a
-            >
-          </div>
-          <div class="col-lg-4 col-md-12">
-            <button
-              class="btn btn-block btn-primary"
-              (click)="displayDecisionForm(approvalDecisionValue.REJECT)"
-            >
-              {{ 'orderApproval.showForm_REJECT' | cxTranslate }}
-            </button>
-          </div>
-          <div class="col-lg-4 col-md-12">
-            <button
-              class="btn btn-block btn-primary"
-              (click)="displayDecisionForm(approvalDecisionValue.APPROVE)"
-            >
-              {{ 'orderApproval.showForm_APPROVE' | cxTranslate }}
-            </button>
-          </div>
+    </ng-container>
```

Comment:
Need to re-save this file because travis is complaining about prettier 
- maybe your prettier was not properly working at the time

```
projects/storefrontlib/src/cms-components/organization/order-approval/order-approval-detail/order-approval-detail-form/order-approval-detail-form.component.html
```

Maybe save the spec file as well

Reply:
Thanks! Fixed.
I should have caught this earlier.